incubator/patroni/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: patroni
 description: 'Highly available elephant herd: HA PostgreSQL cluster.'
-version: 0.9.0
+version: 0.9.1
```

Comment:
I believe the warrants more than just a micro/patch bump.

Reply:
Ok, changed to 0.10.0